lib/responses/token_info.dart

```diff
@@ -21,7 +20,7 @@ class TokenInfo extends Response {
   @JsonKey(nullable: false)
   final TokenType type;
   @JsonKey(nullable: false)
-  final String token;
+  final dynamic token;
```

Comment:
what is dynamic?
couldn't understand why this has to change from String

Reply:
https://github.com/pokepay/flutter-sdk/blob/bf99f02bce497badb9969a4dcf716155845fed40/lib/pokepay_sdk.dart#L185

Because when the token is from `Bill`, `Check` and `CPM`, this will be  `Bill`, `Check` and `CPM` separately; if the token is from `Cashtray`, `PAYREGI` or unknown, this will be an empty string.

Reply:
So its String or empty, right?
`dynamic` is more like a wildcard so if its possible you shouldn't use this type.

Reply:
No, it will be String, `Bill`, `Check` or `CPM`, like in [Android](https://github.com/pokepay/android-sdk/blob/baa47f8509fe5c77c41f9cb63ad37c9fb8e9850d/pokepaylib/src/main/java/jp/pokepay/pokepaylib/TokenInfo.java#L13) and [iOS](https://github.com/pokepay/ios-sdk/blob/master/Sources/Pokepay/Responses/TokenInfo.swift)

Reply:
@WeiWW  so its not a token string itself, but a TokenInfo Object (or empty String)?
It could be better if we could use like a type `TokenInfo?`.  (because empty string means nothing and types like dynamic or any is a little smelly code, you know)

Reply:
@taniryo-kun I think this would be open another issue for this because currently both [Android](https://github.com/pokepay/android-sdk/blob/baa47f8509fe5c77c41f9cb63ad37c9fb8e9850d/pokepaylib/src/main/java/jp/pokepay/pokepaylib/Pokepay.java#L85) and [iOS](https://github.com/pokepay/ios-sdk/blob/6b8cc37c63c3754d48fa59a36f5043ac27fdfd6c/Sources/Pokepay/Pokepay.swift#L100) are doing the same way.